java/benchmarks/src/main/java/glide/benchmarks/utils/Benchmarking.java

```diff
@@ -201,6 +228,8 @@ public static void testClientSetGet(
                     printResults(calculatedResults, (after - started) / NANO_TO_SECONDS, iterations);
                 }
             }
+            executor.shutdownNow();
+            threadPoolResource.getEventLoopGroup().shutdownGracefully();
```

Comment:
`shutdownGracefully()` returns a future. Should we wait for it?

Reply:
Sure. Not a bad idea.